docs/sources/tempo/operations/best-practices.md

```diff
@@ -21,23 +21,42 @@ In the screenshot below, the left side of the screen (1) shows the list of resul
 These can also include information on the libraries that were used to instrument the spans for a trace, or any other infrastructure information.
 
 For more information, read the [Attribute and Resource](https://opentelemetry.io/docs/specs/otel/overview/) sections in the OpenTelemetry specification.
 
 ### Naming conventions for span and resource attributes
 
+The OpenTelemetry project defines a number of semantic conventions for attributes, which can help you to determine which attributes are most important to include in your spans. These conventions provide a common vocabulary for describing different types of entities, which can help to ensure that your data is consistent and meaningful.
+
 When naming attributes, use consistent, nested namespaces to ensures that attribute keys are obvious to anyone observing the spans of a trace and that common attributes can be shared by spans.
 Using our example from above, the `http` prefix of the attribute is the namespace, and `url` and `status_code` are keys within that namespace.
 Attributes can also be nested, for example `http.url.protocol` might be `HTTP` or `HTTPS`, whereas `http.url.path` might be `/api/v1/query`.
 
-For more details around semantic naming conventions, refer to the [Recommendations for OpenTelemetry Authors](https://opentelemetry.io/docs/specs/otel/common/attribute-naming/#recommendations-for-opentelemetry-authors) documentation.
+For more details around semantic naming conventions, refer to the [Recommendations for OpenTelemetry Authors](https://opentelemetry.io/docs/specs/otel/common/attribute-naming/#recommendations-for-opentelemetry-authors) and [OpenTelemetry Semantic Conventions](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/README.md) documentation.
 
 Some third-party libraries provide auto-instrumentation that generate span and span attributes when included in a source base.
 
 For more information about instrumenting your app for tracing, refer to the [Instrument for distributed tracing](/docs/tempo/latest/getting-started/instrumentation/) documentation.
 
+### How many attributes should spans have?
+
+The decision of how many attributes to include in your spans is up to you, there is no hard and fast rule.
+Keep the number of attributes to a minimum, as each attribute adds overhead to the tracing system.
+
+Only include attributes that are relevant to the operation that the span represents. For example, if you are tracing an HTTP request, you might include attributes such as the request method, the URL, and the response status code.
+
+If you are unsure whether or not to include an attribute, it is always better to err on the side of caution and leave it out. You can always add additional attributes later if you need them.
+
+In general, consider the following guidelines:
+
+- Don't include metrics or logs as attributes in your spans.
```

Comment:
i actually want people to start putting logs into spans but that's another discussion entirely. this is likely good advice :)

Reply:
Then we'll have a fun update -- that could be an interesting blog post opening: "General wisdom usually says don't include metrics or logs as attributes in your spans.

Unless you work in Tempo!"